javascript/javascript3/week2/fetching-exercise/Fetching_waiting.js

```diff
@@ -0,0 +1,33 @@
+console.log("script loaded");
+
+new Promise(function(resolve, reject) {
+    setTimeout(() => {
+        resolve("Whoops!");
+    }, 3000);
+})
+
+fetch("http://api.open-notify.org/astros.json")
+    // .then below runs when the remote server responds
+    .then(function(response) {
+        // response.text() returns a new promise that resolves with the full response text
+        // when it loads
+        return response.text();
+    })
+    .then(function(text) {
+        // ...and here's the content of the remote file
+        alert(text); // {"name": "iliakan", "isAdmin": true}
+    });
+
+
+// async way
+async function getData() {
+    try {
+        const h1Element = document.getElementsByTagName('h1')
+        const response = await fetch("http://api.open-notify.org/astros.json");
+        const stringify = await response.json();
```

Comment:
The name of the constant is not correct. First of all its a verb. Verbs are usually used for functions. Like fetch()
Secondly, it holds an object, not a string